app/jsx/pages/ItemPage.jsx

```diff
@@ -50,14 +55,17 @@ class ItemPage extends PageBase {
 
   state = { currentTab: this.articleHashHandler(this.tabNameFromHash()) }
 
-  componentDidMount() {
+  componentDidMount(...args) {
     // Check hash whenever back or forward button clicked
     window.onpopstate = (event) => {
       var h = this.articleHashHandler(this.tabNameFromHash())
       if ((h != this.state.currentTab) && anchors.includes(h)) {
         this.setState({currentTab: h})
       }
     }
+    // This is overriding PageBase componentDidMount
+    // ToDo: https://medium.com/@dan_abramov/how-to-use-classes-and-sleep-at-night-9af8de78ccb4
```

Comment:
Just read that article. Food for thought. I'd be interested in discussing with you how we could use composition instead of inheritance for the stuff that PageBase provides. Never gave it thought until now.